### ISSUE TITLE
not an issue

### ISSUE THREAD

**Issue by neenabastin** [eneim/Toro#206]
I would like to get a solution,
I would like to insert dailymotion view along with the native player view in recyclerview, my current project have a requirement like this. Can you please help on this?

**Comment by eneim** [comment]
@neenabastin What is the "dailymotion view"? They have an SDK with their own Player?

**Comment by neenabastin** [comment]
Yes they have a dailymotion webview component which we can add in xml file and their own sdk also. I would like to inco-operate both together, Since I have to support dailymotion videos and bitgravity videos
